tomcat/src/main/java/org/apache/coyote/http11/request/RequestBody.java

```diff
@@ -16,31 +18,25 @@ public class RequestBody {
 
     private final Map<String, String> items = new HashMap<>();
 
-    private RequestBody() {
+    public RequestBody() {
+        this(Collections.emptyMap());
     }
 
     private RequestBody(final Map<String, String> items) {
         this.items.putAll(items);
     }
 
-    public static RequestBody empty() {
-        return new RequestBody();
-    }
-
     public static RequestBody from(final String body) {
-        if (body.isEmpty()) {
-            return new RequestBody();
-        }
         return Arrays.stream(body.split(SEPARATOR))
                 .map(field -> field.split(DELIMITER))
                 .collect(collectingAndThen(
```

Comment:
현재 로그인과 회원가입의 입력칸에 아무것도 넣지 않았을 때
해당 부분의 로직에서 
`index에 해당하는 value가 없으면 IndexOutOfBoundError 발생 -> Http11Processor 에서 catch -> 500 Response 반환`
의 흐름을 통해 처리하고 있는데요.


```suggestion
        return Arrays.stream(body.split(SEPARATOR, 2))
                .map(field -> field.split(DELIMITER, 2))
                .collect(collectingAndThen(
```
의 형태로 바꾸어 빈 문자열을 받은 뒤 이에 따른 예외 처리를 진행해 주는 방식은 어떤가요?

Reply:
아 그러네요 ㅎㅎ 잘못 봤습니다.!